### PR TITLE
ThreeSum

### PR DESCRIPTION
**Pull request by vidhya001**
added code to issue #163 ThreeSum with example ,complexity analysis

### CODE REVIEW

File: Code/C++/ThreeSum.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  cin>>nums[i];
+  }
+  threeSum(nums);
+
+}
+
+
+
+/*
+Example 1:
```
Comment (rudrakshi99) [inline Code/C++/ThreeSum.cpp:75]: add at least 2 test cases.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

File: Code/C++/ThreeSum.cpp (hunk 1 above)
Comment (vidhya001) [reply Code/C++/ThreeSum.cpp:75]: yes sure

**Review by vidhya001 — COMMENTED** [review]
Added examples

File: Code/C++/ThreeSum.cpp [hunk 2]
```diff
@@ -85,6 +85,7 @@ Example 3:
 Input: nums=[1,-1] 
 Output: No such elements found
 
+
 Complexity Analysis:
 Time complexity: O(N^2).
```
Comment (rudrakshi99) [inline Code/C++/ThreeSum.cpp:90]: add space complexity also.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Review by rudrakshi99 — APPROVED** [review]

**Comment by rudrakshi99** [comment]
update the readme in the array folder.

**Comment by rudrakshi99** [comment]
Resolve the merge conflicts.

**Review by vidhya001 — COMMENTED** [review]
I have solved the merge conflicts

**Comment by rudrakshi99** [comment]
> I have solved the merge conflicts

It's still there.

**Comment by tarun26091999** [comment]
@vidhya001 link the code in "Three Sum" section. Do not create new section named "3 sum"

**Comment by vidhya001** [comment]
@tarun26091999, Are there any changes that need to be done?
I have seen @rudrakshi99 approved my changes.
So if there any changes needed from my side please let me know.

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
make indentations correct.

**Review by vidhya001 — COMMENTED** [review]
I have changed the file name to Three_Sum.cpp making it similar to that of python. If there any changes to be done, please let me know @tarun26091999

**Comment by rudrakshi99** [comment]
> Resolve merge conflicts.

Please do again.

**Pull request merged**